backend/Dockerfile

```diff
@@ -51,22 +53,32 @@ FROM base as development
 # local filesystem which will need a rebuild anyway
 
 # Run command
-# (for development we need to execute npm install since the
+# (for development we need to execute yarn install since the
 #  node_modules are on another volume and need updating)
-CMD /bin/sh -c "yarn install && yarn run dev"
+CMD /bin/sh -c "cd /database && yarn install && yarn build && cd /app && yarn install && yarn run dev"
```

Comment:
May be we have to copy the package.json of database to the newly created folder?